frontend/public/components/build.tsx

```diff
@@ -165,6 +167,22 @@ const BuildGraphs = requirePrometheus(({ build }) => {
   );
 });
 
+export const PipelineBuildStrategyAlert: React.FC<BuildsDetailsProps> = ({ obj: build }) => {
```

Comment:
This shouldn't need to take in any props

```suggestion
export const PipelineBuildStrategyAlert: React.FC = () => {
```

Reply:
removed